profiles/mukul314.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/76463001/190381823-6023c142-cf15-45b7-9117-a19fc9deb098.png",
+  "issueId": 66,
+  "name": "Eddie Jaoude",
+  "username": "eddiejaoude"
```

Comment:
```suggestion
  "image": "https://user-images.githubusercontent.com/76463001/190381256-a7bc8323-82c0-4d7e-9a37-d3c4b2a8de14.png",
  "issueId": 1379,
  "name": "Mukul Ojha",
  "username": "mukul314"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Thanks for the suggestion man.
I almost forgot to change the other things, I didn't notice them :))